docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -714,6 +714,12 @@ connector using a {doc}`WITH </sql/create-table-as>` clause.
 * - `orc_bloom_filter_fpp`
   - The ORC bloom filters false positive probability. Requires ORC format.
     Defaults to `0.05`.
+* - `sorted_by`
```

Comment:
Move this above after `partitioning`. We leave these ORC properties at the end since they are less general.